docs/installation.rst

```diff
@@ -155,14 +155,18 @@ point to the root article:
 Include urlpatterns
 ~~~~~~~~~~~~~~~~~~~
 
-To integrate the wiki to your existing application, you should add the
-following lines at the end of your project's ``urls.py``.
+To integrate the wiki to your existing application, you should ensure the
+following lines in your project's ``urls.py``.
```

Comment:
Nitpick, but I think this reads a bit odd.

How about:
>To integrate the wiki in your existing application, you should ensure the following lines are included in your project's ``urls.py``. 

My English isn't perfect so this might not be any better. :upside_down_face:

Reply:
Thanks, you are absolutely right :)